torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1985,6 +1985,51 @@ def test_return_local_rrefs(self):
 
         self.assertEqual(rets, [11, 12, 13])
 
+    @dist_init
+    def test_rref_type(self):
+
+        def launched_rpc(events):
+            expected_name = "rpc_sync#_rref_typeof_on_owner"
+            return any([e.name.startswith(expected_name) for e in events])
+
+        dst = worker_name((self.rank + 1) % self.world_size)
+        rref = rpc.remote(dst, torch.add, args=(torch.ones(2), 1))
+
+        with torch.autograd.profiler.profile() as p:
+            t = rref._get_type()
+
+        print(p.function_events)
+        self.assertTrue(launched_rpc(p.function_events))
+        self.assertEqual(t, type(torch.ones(2)))
+
+        with torch.autograd.profiler.profile() as p:
+            for _ in range(10):
+                t = rref._get_type()
```

Comment:
nit: t is not used?

Reply:
I should have added another `self.assertEqual(t, type(torch.ones(2)))` to verify that the type is still the same.